src/BitmapIO.cpp

```diff
@@ -39,6 +39,26 @@ static bool is_bmp(Gosu::Reader reader)
     return magic_bytes[0] == 'B' && magic_bytes[1] == 'M';
 }
 
+/// Returns a copy of the given Gosu::Image with the alpha channel removed.
+/// Every pixel with alpha==0 will be replaced by Gosu::Color::FUCHSIA.
+static std::vector<Gosu::Color::Channel> bitmap_to_rgb(const Gosu::Bitmap& bmp)
```

Comment:
Required to work around the fact that stb_image_write will now actually write BMP files with an alpha channel: https://github.com/nothings/stb/commit/b14a80784c689979b75f6e8929a7d7b4b4a86678

...and for compatibility reasons, Gosu should keep writing 24-bit BMP files